winml/lib/Api/ImageFeatureDescriptor.h

```diff
@@ -94,3 +105,9 @@ struct ImageFeatureDescriptor : ImageFeatureDescriptorT<
   ImageColorSpaceGamma color_space_gamma_;
 };
 }  // namespace winrt::Windows::AI::MachineLearning::implementation
+
+namespace winrt::Windows::AI::MachineLearning::factory_implementation {
+    struct ImageFeatureDescriptor : ImageFeatureDescriptorT<ImageFeatureDescriptor, implementation::ImageFeatureDescriptor> {
```

Comment:
Can you help me understand what this is for?

Reply:
these factories allow objects to be created.  prior to this change, external people could not create objects.   this change allows the creation of these objects.   factories are things that create objects.

---
In reply to: [344928039](https://github.com/microsoft/onnxruntime/pull/2368#discussion_r344928039) [](ancestors = 344928039)

Reply:
Thanks! That makes sense